src/utils/pgwire/src/pg_extended.rs

```diff
@@ -804,5 +806,25 @@ mod tests {
                 "'2021-01-07 06:13:20'::TIMESTAMP"
             ]
         );
+
+        let mut raw_params = vec![BytesMut::new(); 2];
```

Comment:
We can add some comments to indicate what feature we are testing or abstract into some small test function with meaningful name.